app/assets/stylesheets/components/_wallet.scss

```diff
@@ -179,7 +179,8 @@
     padding-bottom: 1em;
 }
 
-.wallet-create, .create-account-wrapper {
+.wallet-create,
```

Comment:
Please double check all files when committing, this is a reformatting only commit